default.nix

```
@@ -182,6 +182,10 @@ let
     name = "hnix";
     root = ./.;
 
+    overrides = self: super: {
+      neat-interpolation = pkgs.haskellPackages.neat-interpolation_0_5_1;
```

```suggestion
      neat-interpolation = super.neat-interpolation_0_5_1;
```

Seems to fix the problem of it not being picked up.

Only getting one test failure now:
```
    indented antiquotation:                                          FAIL
      tests/PrettyTests.hs:38:
      When pretty-printing Fix (NStr (Indented 0 [Plain "echo $foo"]))
      expected: "''echo $foo''"
       but got: "  ''\n  echo $foo\n  ''"
```
Maybe you've managed to fix the newlines issue as well :) ? :+1: 

Btw this one also works
```suggestion
      neat-interpolation = haskellPackages.neat-interpolation_0_5_1;
```
as it picks `haskellPackages = haskell.packages.${compiler}` defined above instead of the default `pkgs.haskellPackages` set.

---

Awesome! :)